diff --git a/node_modules/@storybook/angular/dist/ts3.9/server/angular-cli-webpack-13.x.x.js b/node_modules/@storybook/angular/dist/ts3.9/server/angular-cli-webpack-13.x.x.js
index 77b40b2..9e3d9af 100644
--- a/node_modules/@storybook/angular/dist/ts3.9/server/angular-cli-webpack-13.x.x.js
+++ b/node_modules/@storybook/angular/dist/ts3.9/server/angular-cli-webpack-13.x.x.js
@@ -54,7 +54,7 @@ var __spreadArrays = (this && this.__spreadArrays) || function () {
 };
 // Private angular devkit stuff
 var generateI18nBrowserWebpackConfigFromContext = require('@angular-devkit/build-angular/src/utils/webpack-browser-config').generateI18nBrowserWebpackConfigFromContext;
-var _a = require('@angular-devkit/build-angular/src/webpack/configs'), getCommonConfig = _a.getCommonConfig, getStylesConfig = _a.getStylesConfig, getTypescriptWorkerPlugin = _a.getTypescriptWorkerPlugin;
+var _a = require('@angular-devkit/build-angular/src/webpack/configs'), getCommonConfig = _a.getCommonConfig, getStylesConfig = _a.getStylesConfig, getTypescriptWorkerPlugin = _a.getDevServerConfig;
 var TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 var filterOutStylingRules = require('./utils/filter-out-styling-rules').filterOutStylingRules;
 /**
